test/performance/sparse/domainAssignment.chpl

```diff
@@ -0,0 +1,115 @@
+use Time;
+
+use LayoutCS;
+
+var t = new Timer();
+
+config const correctness = true;
+config const numIndices = 1000;
+config const sparsity = 0.01;
+
+const numIndicesInParentDom = numIndices/sparsity;
+const parentSize = sqrt(numIndicesInParentDom):int;
+const parentDom = {1..parentSize, 1..parentSize};
+const sparseStride = (1/sqrt(sparsity)):int;
+const sparseIndices = {1..parentSize by sparseStride,
+                       1..parentSize by sparseStride};
+
+proc startDiag() {
```

Comment:
What is the meaning of `startDiag`/`stopDiag` in this context? Would `{start,stop}Timer` make more sense?

Reply:
That was to avoid storing times in different variables and printing out at the end and making those small blocks more or less self-contained. I don't mind changing it, though.